reports/urls.py

```diff
@@ -5,5 +5,4 @@
 urlpatterns = [
     path('', ReportView.as_view(), name='get_xlsx_reports'),
     path('data_export/', data_export, name='get_data_export'),
-    path('wazimap', WazimapView.as_view(), name='wazimap'),
```

Comment:
Don't remove it, redirect it to `/genmap`

Reply:
Sure we can do that. Although, I feel like just letting Jean and Sarah know about the new URL would be easier than having this duplicate.